### DISCUSSION TITLE
How many time needed when you deploy to vercel ? Did you crash for SSG?

### DISCUSSION THREAD

**Discussion by 6174** [morethanmin/morethan-log#98] [Q&A]
for [slug].jsx ssg in vercel deploy , my situation is always crash. I check the code, every article need to query whole database of notion. So it take too much time. 

Want to know how you can successful build these pages?

**Comment by morethanmin** [comment]
Refer to the picture below for the time taken when deploying to vercel.

![image](https://user-images.githubusercontent.com/72514247/218084173-fbcb4b2e-45df-4a96-909f-1e3ecbf34800.png)

Also can you explain a bit more about crash? Build time could probably be shortened!
